images/scripts/rhel.setup

```diff
@@ -184,6 +184,12 @@ fi
 
 dnf install -y $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
 
+# We are going to install the newest selinux-policy into mock and we need at least that version
+# Always update to prevent case where image has older version than mock
+if [ "${IMAGE#rhel-9*}" != "$IMAGE" ]; then
```

Comment:
Why is this specific to RHEL 9? Doesn't it in principle apply to RHEL 8/CentOS (we just don't build the policy there for now) and Fedora (fedora.setup) as well?

More importantly, I think we should always run `dnf update -y` on all RHEL/Fedora images to make sure we don't fall behind if images are not built often enough.

Reply:
> Why is this specific to RHEL 9?

As I wrote above, this was test (and quick fix). We don't need it on rhel-8/centos.

> More importantly, I think we should always run dnf update -y on all RHEL/Fedora images to make sure we don't fall behind if images are not built often enough.

I was thinking if we should not update those images in any case, but  did not want to introduce this kind of change in this PR.

Reply:
Fully agreed, this was just thinking aloud.